Magic/pom.xml

```diff
@@ -557,6 +557,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
```

Comment:
We have discussed updating to Java 8, I probably will do it as part of the 1.13 compatibility release, but I think it should be a separate change, and not part of this PR.

Reply:
Odd, I thought I left that out of the PR. At one point the collection of the aggregate parameters used streams but I reverted that change. I guess IntelliJ doesn't revert the pom.